pdc_client.egg-info/PKG-INFO

```diff
@@ -0,0 +1,16 @@
+Metadata-Version: 1.1
```

Comment:
@ahills; the `pdc_client.egg-info` folder and the files which under this folder, I consider they are not necessary in this patch. Do you have other purpose to add this folder?